pages.pt_PT/common/yt-dlp.md

```diff
@@ -0,0 +1,37 @@
+# yt-dlp
+
+> Um fork do youtube-dl com funcionalidades e correções adicionais.
+> Descarrega vídeos do YouTube e de outros websites.
+> Mais informações: <https://github.com/yt-dlp/yt-dlp>.
+
+- Descarregar um vídeo ou playlist (com as predefinições do comando abaixo):
+
+`yt-dlp "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
+
+- Descarregar um vídeo com um formato definido. Neste caso, fundindo o melhor formato de vídeo com o melhor formato de áudio (Predefinição):
```

Comment:
Pequeno ajuste de texto e formatação:

```suggestion
- Descarregar um vídeo num formato específico; neste caso, fundindo o melhor formato de vídeo com o melhor formato de áudio (predefinição):
```

Reply:
Também acho estranho o "(predefinição)". Isso significa que essa fusão dos melhores formatos é o que seria aplicado sem se especificar o formato? Sendo assim, qual a utilidade de demonstrar esta opção? Mais valia usar parâmetros não predefinidos neste exemplo, o que seria mais realista.

(Eu sei que esta página replica o que está em inglês; a ideia seria depois aplicar a mesma correção ao inglês, mas se preferires podemos abrir um novo issue para discutir isso separadamente, e depois concluir este PR com a resolução que for decidida lá.)

Reply:
> Também acho que está confuso.

Tens alguma sugestão de uma opção que seria não-predefinida mas com uma descrição simples/curta? 

> Também seria a favor de mudar o exemplo na versão inglesa.

Sim, claro, a ideia seria sincronizar depois. Podemos fazer a alteração aqui e depois sugerir a alteração à página em Inglês (ou até fazer já neste PR a alteração!)

Reply:
Talvez à semelhança que temos:

```
- Especificar o formato de áudio extraído (a predefinição é "best"):

`yt-dlp --extract-audio --audio-format {{mp3}} "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
```

Também poderíamos ter o seguinte, para manter consistente?

```
- Descarregar um vídeo num formato específico (a predefinição é "bv*+ba/b"); neste caso, escolhe o melhor vídeo mp4 disponível:

`yt-dlp --format "{{bv*[ext=mp4]+ba[ext=m4a]/b[ext=mp4]}}" "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
```

Fui buscar o exemplo à [documentação do yt-dlp](https://github.com/yt-dlp/yt-dlp#format-selection-examples).

Reply:
The suggestion above was applied in [`990b2ba`](https://github.com/tldr-pages/tldr/pull/8936/commits/990b2bae12868d09540209c19918e5a761508799). The additional change discussed in this thread was applied in [`7b15476`](https://github.com/tldr-pages/tldr/pull/8936/commits/7b15476ca39f71c5417417a84fcc5f532ff393f3).